internal/api/attestationconfigapi/fetcher_test.go

```diff
@@ -128,13 +127,9 @@ func (f *fakeConfigAPIHandler) RoundTrip(req *http.Request) (*http.Response, err
 		res.Body = io.NopCloser(bytes.NewReader(bt))
 		res.StatusCode = http.StatusOK
 		return res, nil
-
 	} else if req.URL.Path == fmt.Sprintf("/constellation/v1/attestation/azure-sev-snp/%s.sig", f.latestVersion) {
 		res := &http.Response{}
-		obj := AzureSEVSNPVersionSignature{
-			Signature: signature,
-		}
-		bt, err := json.Marshal(obj)
+		bt, err := json.Marshal(nil)
```

Comment:
`json.Marshal(nil)` just returns the string `null`.
You could replace the "logic" here with the hardcoded value.

Reply:
imo `bt` is to short as identifier here, unclear what it should stand for at the first glance.